tox.ini

```diff
@@ -26,6 +28,7 @@ setenv =
 deps =
     -e.[test]
     psycopg2-binary
+    django22: Django>=2.2.17,<3.0
```

Comment:
fyi - doesn't support python 3.9 below 2.2.17 and both GSC and SO are above that one.